analyzer/tools/build-logger/Makefile.manual

```diff
@@ -65,18 +65,34 @@ pack64bit: 64bit packbin
 	done
 	rm -f ldlogger_64.so
 
+pack64bit_only: 64bit_only packbin
+	for x8664dir in 'x86_64'; do \
+		mkdir -p $(LIB_DIR)/$$x8664dir ; \
+		cp ldlogger_64.so $(LIB_DIR)/$$x8664dir/ldlogger.so ; \
+	done
+	rm -f ldlogger_64.so
+
 # pack binary
 packbin:
 	mkdir -p $(BIN_DIR)
 	cp ldlogger $(BIN_DIR)
 	rm -f ldlogger
 
+# pack binary
+packbin:
```

Comment:
We have the same target name above this target. We should rename this one to packbin_64 or similar.